ReSharper.FSharp/test/data/features/quickFixes/removeNeverMatchingRule/Multiple rules 01.fs

```diff
@@ -0,0 +1,8 @@
+module Module
+
+do
+    match () with
+    | _ -> ()
+    | _ -> (){caret}
+    | _ -> ()
```

Comment:
Nit pick: I'd probably use some named patterns to add some clarity to what is deleted and what is remained.